pkg/network/namescheme/networknamescheme_test.go

```diff
@@ -23,14 +23,16 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
 	virtv1 "kubevirt.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/pkg/network/namescheme"
 )
 
 var _ = Describe("Network Name Scheme", func() {
-	Context("CreateNetworkNameScheme", func() {
-		DescribeTable("should return the expected NetworkNameSchemeMap",
+	Context("CreateNetworkNameScheme", func() { //nolint:dupl
```

Comment:
Per my understanding, the linter errors here because it detect 40 lines of duplication in this Context body.

I do not feel comfortable to approve overriding the linter.
One possible solution is to pass to the table the factory function and in the entry to describe that it handles an ordered or hashed scenario.